pages/common/hledger-web.md

```diff
@@ -0,0 +1,32 @@
+# hledger-web
+
+> A robust, friendly plain text accounting app (the web UI & API).
```

Comment:
```suggestion
> Web interface and API to `hledger`.
> A robust, friendly plain text accounting app.
```
Maybe, we can separate the descriptions to highlight the web UI and API part from the generic description.